packages/jest-environment-puppeteer/src/readConfig.js

```diff
@@ -48,4 +49,17 @@ async function readConfig() {
   return merge({}, defaultConfig, localConfig)
 }
 
-export default readConfig
+export function getPuppeteer(config) {
+  switch (config.browser.toLowerCase()) {
+    case 'chromium':
+      // eslint-disable-next-line global-require, import/no-dynamic-require, import/no-extraneous-dependencies
+      return require('puppeteer')
+    case 'firefox':
+      // eslint-disable-next-line global-require, import/no-dynamic-require, import/no-extraneous-dependencies
+      return require('puppeteer-firefox')
+    default:
+      throw new Error(
+        `!!env variable JEST_PUPPETEER_BROWSER is given unsupported value: ${browser}`,
```

Comment:
I think this message is not good. I don't see any `JEST_PUPPETEER_BROWSER` variable elsewhere.

Reply:
Fixed.